CIP-1694/README.md

```diff
@@ -637,26 +635,34 @@ In particular, it is possible to elect an empty committee if the community wishe
 There will be a new protocol parameter for the minimal size of the committee,
 itself a nonnegative number.
 
-#### Term limits
+#### Terms
 
-Each newly elected constitutional committee will have per-member term limits.
-Per-member limits allow for a rotation scheme, such as a third of the committee
+Each newly elected constitutional committee will have a term.
+Per-member terms allow for a rotation scheme, such as a third of the committee
 expiring every year.
 Expired members can no longer vote.
 Member can also willingly resign early, which will be marked on-chain as an expired member.
 
 The system will automatically enter a state of no-confidence when the number of non-expired
 committee members falls below the minimal size of the committee.
-For example, a committee of size five with a quorum of three and two expired members can still
-pass governance actions if all of non-expired members vote `Yes`.
+For example, a committee of size five with a threshold of 3/5 a minimum size of three and two expired members can still
+pass governance actions if two non-expired members vote `Yes`.
 However, if one more member expires then the system enters a state of no-confidence,
 since the two remaining members are not enough to meet quorum.
 
-The maximum term limit is a governance protocol parameter, specified as a number of epochs.
+The maximum term is a governance protocol parameter, specified as a number of epochs.
 During a state of no-confidence, no action can be ratified,
 so the committee should plan for its own replacement if it wishes to avoid disruption.
 
-<!--------------------------- Constitutional committee ------------------------>
+#### Proposal policy
+
+While the constitution is an informal, off-chain document, there will
+also an optional script that can enforce some guidelines. This script
+acts to supplement the constitutional committee by restricting some
+proposal types. For example, if the community wishes to have some hard
+rules for the treasury that cannot be violated, a script that enforces
+these rules can be voted in as the proposal policy.
```

Comment:
Yes, that would be another example. A more flexible version would be to have formal committees with a token to prove membership. A more obviously desirable thing would be to put hard bounds on certain parameters.

The constitution was the most natural thing to include it with. But I agree that it might be better as its own thing. The only issue is that adding a governance action is a bunch of work that affects a lot of existing code/other material.